oauthenticator/tests/test_google.py

```diff
@@ -66,3 +66,40 @@ def google_client(client):
     with raises(HTTPError) as exc:
         name = await authenticator.authenticate(handler)
     assert exc.value.status_code == 403
+
+
+async def test_user_in_groups(google_client):
+    authenticator = GoogleOAuthenticator(
+        hosted_domain=['email.com', 'mycollege.edu'],
+        admin_google_groups={'email.com': ['fakeadmingroup']},
+        google_group_whitelist={'email.com': ['fakegroup']}
+    )
+    admin_handler = google_client.handler_for_user(user_model('fakeadmin@email.com'))
+    admin_user_info = await authenticator.authenticate(admin_handler, google_groups=['anotherone', 'fakeadmingroup'])
+    user_is_admin = admin_user_info['admin']
+    assert user_is_admin == True
+    whitelist_handler = google_client.handler_for_user(user_model('fakewhitelisted@email.com'))
+    whitelist_user_info = await authenticator.authenticate(whitelist_handler, google_groups=['anotherone', 'fakegroup'])
+    user_is_whitelisted = whitelist_user_info['auth_state']['google_user']['google_groups']
+    user_is_not_admin = whitelist_user_info['admin']
+    assert 'fakegroup' in user_is_whitelisted
+    assert user_is_not_admin == False
```

Comment:
There's no need to name the vars based on their expected value.
For example having a var named `user_is_not_admin` and then doing this assert:
```
assert user_is_not_admin == False
```
From the naming, it implies that the user is an admin (and I think the intention was different).
I believe it would be more clear if we have just a var `admin_user` and then asserting:
``` python
# In the first scenario when the user is part of the admin group
assert admin_user == True
```
```python
# In the second scenario when the user is not part of the admin group
assert admin_user == False
```
Same for:
* `user_is_whitelisted` and `non_whitelist_user_info`: could both be `whitelisted_user_groups`,
* `admin_field_does_not_exist`: could be `admin_field`
* the handlers

Having long names for the variables or naming them based on their truthfulness can become a little confusing. What do you think @missingcharacter?

Reply:
done!